DigitalLearningSolutions.Web/Helpers/RegistrationEmailValidator.cs

```diff
@@ -85,20 +84,13 @@ ICentresService centresService
                         DuplicateEmailErrorMessage
                     );
                 }
-                else if (isRegisterAdminJourney && primaryEmailMatchesCentre == false)
+                else if (isRegisterAdminJourney && model.PrimaryEmail != autoRegisterManagerEmail &&
+                         model.SecondaryEmail != autoRegisterManagerEmail)

```

Comment:
I think we have lost the case-insensitivity here.

Reply:
Ah right, added it back